help/en/html/doc/Technical/ReleaseProcess.md

```diff
@@ -0,0 +1,187 @@
+ - Publish from `dev-minor` roughly 4-6 weeks and strongly encourage use to get feedback; significant problems can be fixed via `dev-update`
+ - Publish from `dev-major` on targeted dates with major changes.
+
+To it another way:  Publish the `dev-minor` branch to users often and consistently, using the `dev-update` to get fixes out more often.  This is basically "test releases with fixes". Eventually, effort focuses on one set of fixes until that particular head of `dev-update` is good enough to call a production release.
+
+One could also imagine a "periodic production release from `dev-major` with fixes" model, but the key question remains:  How do we use these numbers to simultaneously get fixes to users _and_ encourage them to test new features?
+
+## Web Contents
+
+The default web content (from [https://jmri.org](https://jmri.org)) will be from the HEAD of the `dev-update` branch.  This allows minor content fixes to get on the web immediately.
+
+Simultaneously, the HEAD of the `dev-minor` and `dev-major` branches will populate web content starting at [https://jmri.org/minor](https://jmri.org/minor) and [https://jmri.org/major](https://jmri.org/major)  This will allow people to see Javadoc, point people to recently updated pages, etc.
+
+  
+ ## Other Things to Note
+ 
+JMRI has, for a long time, followed a Linux-like release numbering system where odd-numbered minor releases were for development and tests, whilst even numbered releases were for production. That distinction is no longer present here.
+
+This entire system is well suited to "point" releases to fix things.  For example, say the most recent releases have been 4.23.1, 4.24.0 and 5.0.0.  Then
+ - Jim finds a bug and fixes it starting with `master` (same as 4.23.1)
+ - Because it started on `master`, that can be merged anywhere
+ - It's a _bug_, so it can be released as 4.23.2 so that people using that as a long-term thing get the fix with minor disruption.
+ - But it will also be included in a 4.25.0 and 6.0.0 when those get released, so we can decide when to release them and make them available.
+
+It's even possible to make e.g. 4.23.1.1 with _just_ one specific change, and not the other updates that have accumulated toward 4.23.2 since it was created. (We expect this to be very rare, as it in some sense breaks the naming pattern and is likely to cause some confusion)
+
+There is no longer an explicit [deprecation cycle](https://www.jmri.org/help/en/html/doc/Technical/RP.shtml#deprecating). One should certain mark parts of the API as deprecated in a update or minor change; that's polite.  But when the change gets into the major branch, the deprecations will have been removed:  That's part of why the changes are considered "breaking changes". Because we're separately releasing versions that have and don't have breaking changes, users can decide when they want to move forward; effectively managing their own deprecation cycles.
```

Comment:
Does this paragraph mean that I do two PRs close to eachother? First I do a PR that deprecates the items I want to deprecate. And as soon as that PR is merged, I do another PR there I remove the items I deprecated in the previous PR.